### PR TITLE
Lesson 8 and fixed the broken links

### PR DESCRIPTION
**Pull request by CinnamonXI**

### CODE REVIEW

File: lessons/3-NeuralNetworks/05-Frameworks/IntroPyTorch.ipynb [hunk 1]
```diff
@@ -1026,7 +1026,7 @@
       "source": [
         "## Training One-Layer Perceptron\n",
         "\n",
-        "Let's use Tensorflow gradient computing machinery to train one-layer perceptron.\n",
+        "Let's use PyTorch gradient computing machinery to train one-layer perceptron.\n",
```
Comment (CinnamonXI) [inline lessons/3-NeuralNetworks/05-Frameworks/IntroPyTorch.ipynb:1029]: @shwars please confirm this correction is right.

File: lessons/4-ComputerVision/08-TransferLearning/README.md [hunk 2]
```diff
@@ -29,7 +31,11 @@ Let's see transfer learning in action in corresponding notebooks:
...
 
+## [Post-lecture quiz](https://black-ground-0cc93280f.1.azurestaticapps.net/quiz/208)
+
 ## [Lab](lab/README.md)
 
 In this lab, we will use real-life [Oxford-IIIT](https://www.robots.ox.ac.uk/~vgg/data/pets/) pets dataset with 35 breeds of cats and dogs, and we will build a transfer learning classifier.
- 
\ No newline at end of file
+
+> ✅ Todo: there is an unlinked file: [TrainingTricks.md](TrainingTricks.md
```
Comment (CinnamonXI) [inline lessons/4-ComputerVision/08-TransferLearning/README.md:40]: @shwars.

**Comment by CinnamonXI** [comment]
@jlooper please confirm I inserted the quiz links properly. thanks!

**Review by jlooper — APPROVED** [review]
Looks like I messed them up, thank you for the fixes!

**Pull request merged**